Rakefile

```diff
@@ -93,7 +92,6 @@ namespace :cordova do
 
     if platform == "ios"
       Dir.chdir(CORDOVA_PATH) do
-        sh %{ cordova plugin add #{TESTFAIRY_PLUGIN_URL} } if environment == "staging"
```

Comment:
Are we sure? Didn't this add a wrapper that enabled better TestFairy integration such as the ability to prompt to download the latest version on end-user testing mobiles.